### ISSUE TITLE
Why spray's doesn't work?

### ISSUE THREAD

**Issue by Rrhyast** [danielkrupinski/Osiris#3687]
![csgo_FhwIOfGWs3](https://user-images.githubusercontent.com/62977195/170354487-bca54823-8d89-4345-aabc-89742d1808a5.jpg)

**Comment by Patrixsin** [comment]
Cuz they are not implemeted.

**Comment by Rrhyast** [comment]
> Cuz they are not implemeted.

:/ WTF, it's actually available on inventory changer how they're not implemented.

**Comment by OzymoGit** [comment]
its in your invent but not actually useable

**Comment by Patrixsin** [comment]
> :/ WTF, it's actually available on inventory changer how they're not implemented.

In changer there is everything, but it doesnt mean everything works either

**Comment by Rrhyast** [comment]
> > :/ WTF, it's actually available on inventory changer how they're not implemented.
> 
> In changer there is everything, but it doesnt mean everything works either

True, but I saw some Russian inventory changer that has spray changer and sell it for 50 rubles, that's stupid.
